tumblr-gif-search/README.md

```diff
@@ -258,7 +258,7 @@ In this example, we set the number of `parallel` to 8 for `chunk_idx`. After ind
  |-doc_indexer.bin
 ```
 
-You can see that the data is splitted into 8 different directories under the given `workspace`, in a more or less uniform way. This is good because otherwise we end up with one big file that is slow to load and to query. With sharding enabled, one can query multiple smaller index in parallel, which gives better efficiency. 
+You can see that the data is splitted into 8 different directories under the given `workspace`, in a more or less uniform way. This is good because otherwise we end up with one big file that is slow to load and to query. With sharding enabled, one can query multiple smaller index in SHARDS_CHUNK_SEG, which gives better efficiency. 
```

Comment:
here `parallel` is the right word to use

Reply:
I've reverted some of these. Can you have a look again?